packages/tree-view-for-tests/src/index.ts

```diff
@@ -7,6 +7,16 @@ function times(str: string, length: number) {
   return Array.from({ length }, () => str).join('');
 }
 
+// Depending on the file type we either remove carriage returns or not
+// so we get the same hash on windows or mac.
+function agnosticHash(_path: string, fileString: string): string {
```

Comment:
oo ye you're right I could just do it for all files and just update the snapshot.
I think i ended up doing it this way so it would produce the same hashes in the tests as oppose to changing the hashes.

but ye I will remove this logic and update the hashes.